OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -132,6 +146,59 @@ public void onPageLoadProgressChanged(WebView webView, int progress) {
         }
     }
 
+    private void evaluateJavascriptForiFrame() {
```

Comment:
To keep the function more atomic, we could have it return a `Boolean` and on it, we can decide whether to show or hide the banner.